assemblies/assembly_configuring-central/main.adoc

```diff
@@ -10,23 +10,26 @@ include::_artifacts/document-attributes.adoc[]
 include::_artifacts/author-group.adoc[]
 
```

Comment:
The abstract could contain more information and context. The abstract phrasing and format is used across all the docs so I would prefer to leave this as-is for now rather than make changes to a single abstract. I will pose your questions to the team and see what can be done to improve the abstracts.